model_building/xgboost_experiment_configuration.py

```diff
@@ -143,7 +144,7 @@ def initialize_regressor(self):
         if not getattr(self, '_hyperparameters', None):
             self._regressor = xgb.XGBRegressor()
         else:
-            self._regressor = xgb.XGBRegressor(min_child_weight=self._hyperparameters['min_child_weight'], gamma=self._hyperparameters['gamma'], n_estimators=self._hyperparameters['n_estimators'], learning_rate=self._hyperparameters['learning_rate'], max_depth=self._hyperparameters['max_depth'], tree_method="hist", objective='reg:squarederror', n_jobs=1)
+            self._regressor = xgb.XGBRegressor(min_child_weight=self._hyperparameters['min_child_weight'], gamma=self._hyperparameters['gamma'], n_estimators=self._hyperparameters['n_estimators'], learning_rate=self._hyperparameters['learning_rate'], max_depth=self._hyperparameters['max_depth'], tree_method="hist", objective='reg:squarederror', nthread=1, n_jobs=1)
```

Comment:
I'd rather have `nthread=1` only when running parallel experiments

Reply:
I thought that the intended behaviour of the sequential mode was to use a single core for a single experiment, thus the line. I will add the parallelism check

Reply:
Good, thanks! (We should probably let the user choose, but this seems complicated and unimportant, I will deal with it myself in the future)